string name
float64 throttle
float64 steer
float64 brake
int32 drive_mode
